### PR TITLE
State goals and non-goals

### PR DESCRIPTION
**Pull request by mwestphal**
Adress https://github.com/f3d-app/f3d/issues/1239

### CODE REVIEW

**Comment by mwestphal** [comment]
@snoyer @Meakk @pogey @lknknm please review

File: README.md [hunk 1]
```diff
@@ -59,7 +59,29 @@ man f3d # Linux only
...
+
+- Support as many 3D file formats as possible
+- Support many types of visualization
+- Support all industries dealing with 3D datasets
+- Let any user easily and quickly view any model with good defaults
+- Be as modular as possible to be built with a small number of dependencies
+- Be fully controllable from the command line and configuration file
+- Be usable non-interactively
+- Be as configurable as possible
+- Foster and inclusive and beginner friendly community
```
Comment (Meakk) [inline README.md:76]:
```suggestion
- Foster an inclusive and beginner friendly community
```

File: README.md [hunk 2]
```diff
@@ -59,7 +59,29 @@ man f3d # Linux only
 
 F3D is developed by a team of passionate devs. Please use F3D, star it on github or even become a [sponsor](https://github.com/sponsors/f3d-app) to support us!
 
-## Contributing
+# Goals/Non-goals
+
+We believe that clearly stating the goals and non-goals of F3D would help potential users and developers figure out if F3D is the right tool for them.
```
Comment (Meakk) [inline README.md:64]: Is this sentence useful?

Comment (mwestphal) [reply README.md:64]: not that much. I just did not feel right to go directly to goals. I can remove it though.

Comment (Meakk) [reply README.md:64]: Let's wait for other feedback but I'd rather go straight to the point in the main readme.

Comment (snoyer) [reply README.md:64]: Maybe you can skip the whole "goals/non-goals" terminology as this is user facing communication. Could try something less formal like naming the section "Functionality", "Scope", "Vision", or "Philosophy" and simply saying:

> As a minimalist 3D viewer F3D aims to:
> - Support as many 3D file formats as possible
> - ...
>
> but there are no plans to:
> - Provide a classic mouse-based UI, with menus and buttons
> - ...

Comment (mwestphal) [reply README.md:64]: I like that!

File: README.md [hunk 3]
```diff
@@ -59,7 +59,29 @@ man f3d # Linux only
...
+
+## Goals
+
+- Support as many 3D file formats as possible
+- Support many types of visualization
```
Comment (snoyer) [inline README.md:69]:
```suggestion
- Support many types of visualization (meshes, volumic, point clouds, ...)
```
for most people who are not from the sci.viz. community

File: README.md [hunk 4]
```diff
@@ -59,7 +59,29 @@ man f3d # Linux only
...
+- Support all industries dealing with 3D datasets
```
Comment (snoyer) [inline README.md:70]: I know what you mean but "industries" sounds a bit strong/specific; could it be reworded with something more like "use-case"/"scenario"/"workflow"/...?

File: README.md [hunk 5]
```diff
@@ -59,7 +59,29 @@ man f3d # Linux only
...
+- Let any user easily and quickly view any model with good defaults
+- Be as modular as possible to be built with a small number of dependencies
+- Be fully controllable from the command line and configuration file
+- Be usable non-interactively
+- Be as configurable as possible
```
Comment (snoyer) [inline README.md:75]:
```suggestion
- Let any user easily and quickly view any model with good defaults
- Be as configurable as possible
- Be fully controllable from the command line and configuration file
- Be usable non-interactively
- Be as modular as possible to be built with a small number of dependencies
```
reorder with the more "advanced" usage last?

File: README.md [hunk 6]
```diff
@@ -59,7 +59,26 @@ man f3d # Linux only
...
+- Support as many 3D file formats as possible
+- Support many types of renderings (textures, edges, etc... ) and visualizations (meshes, volumic, point sprites)
+- Support any and all use-cases dealing with 3D datasets
+- Let any user easily and quickly view any model with good defaults
+- Be as configurable as possible
+- Be fully controllable from the command line and configuration file
+- Be usable non-interactively
+- Be as modular as possible to be built with a small number of dependencies
+
+but there are no plans to:
```
Comment (Meakk) [inline README.md:75]:
```suggestion
but there is no plan to:
```

**Review by Meakk — APPROVED** [review]

**Pull request merged**